javascript/ditsmod/package.json

```diff
@@ -15,9 +15,9 @@
   "author": "Your Name Here",
   "license": "MIT",
   "dependencies": {
-    "@ditsmod/body-parser": "~2.13.1",
-    "@ditsmod/core": "~2.50.0",
-    "@ditsmod/routing": "~2.0.0"
+    "@ditsmod/body-parser": "*", 
```

Comment:
No, this is a very bad practice that should never be used at all. Please use "~2.13.1".